app/resources/v1/flags.js

```diff
@@ -24,15 +24,15 @@ async function readFlags (res) {
 function sendSuccessResponse (res, flags) {
   res.set({
     'Content-Type': 'application/json; charset=utf-8',
-    'Location': config.restapi.baseuri + '/v1/flags',
+    Location: config.restapi.baseuri + '/v1/flags',
```

Comment:
I didn't realize we'd be as affected as we are by inconsistent quotation marks. I know there's some discussion [here](https://github.com/standard/standard/issues/791).

We could override standard's rule with [eslint's `consistent-as-needed` rule](https://eslint.org/docs/rules/quote-props), but the tradeoffs here are the addition of one more rule override and the need to re-quote entire objects if just one property is introduced that requires quotes.

@whitmanschorn do you have an opinion?

Reply:
I think I'm fine with this way, since it means putting quotes around a key is a good indicator that the key name has a non-standard character. Happy to do it either way, though.

Reply:
OK, we'll take the path of least resistance here.